### ISSUE TITLE
Desktop. Text inside DropdownMenu ignores the parent width on the second open

### ISSUE THREAD

**Issue by igordmn** [JetBrains/compose-jb#792]
Compose 0.5.0-build225

```
import androidx.compose.desktop.Window
import androidx.compose.foundation.clickable
import androidx.compose.foundation.layout.*
import androidx.compose.material.*
import androidx.compose.runtime.*
import androidx.compose.ui.Modifier
import androidx.compose.ui.unit.dp

fun main() = Window {
    var expandedState by remember { mutableStateOf(false) }
    MaterialTheme {
        Button(onClick = {
            expandedState = !expandedState
        }) {
            Text("menu")
        }
        DropdownMenu(expanded = expandedState, onDismissRequest = { expandedState = false }) {
            Box(
                modifier = Modifier
                    .requiredWidth(200.dp)
                    .clickable(
                        onClick = {
                            expandedState = false
                        },
                    )
            ) {
                Text("sed do eiusmod tempor incididunt ut labore et dolore magna aliqua")
            }
        }
    }
}
```

https://user-images.githubusercontent.com/5963351/122430969-b432ad80-cf9c-11eb-9b50-56f26270709f.mp4

Not sure, but probably on the second open we have height = 0 somewhere

**Comment by igordmn** [comment]
Should be fixed in 1.0.0-beta5

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6251">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
